Library/Homebrew/utils/cp.rb

```diff
@@ -0,0 +1,58 @@
+# typed: true
+# frozen_string_literal: true
+
+require "extend/os/cp"
+require "fileutils"
+require "system_command"
+
+module Utils
+  # Helper functions for copying files.
+  module Cp
```

Comment:
```suggestion
  module Copy
```

maybe?

Reply:
That sounds fair given that the module no longer guarantees to use the `cp` shell command by default.

But I think that "cp" is consistent with `FileUtils`'s terminology (which also uses "cp" in method names) and that it more unambiguously denotes file copy rather than e.g. copying in-memory data.

Also, with the module name change, I think we'd need to rename the `force_command` keyword arguments as well. Maybe something like `use_cp_command`?

I'll apply the suggestion if you are fine with these points.

Reply:
> That sounds fair given that the module no longer guarantees to use the `cp` shell command by default.

Yeh 👍🏻 

> But I think that "cp" is consistent with `FileUtils`'s terminology (which also uses "cp" in method names) and that it more unambiguously denotes file copy rather than e.g. copying in-memory data.

I think the `FileUtils` method name isn't great here either but: it's easier to changer this PR than Ruby 😁 

> Also, with the module name change, I think we'd need to rename the `force_command` keyword arguments as well. Maybe something like `use_cp_command`?

Agreed 👍🏻 

Thanks again @tesaguri!

Reply:
Resolved in 028cfe1ea63fd85748219eca53bfda8c0441e661.